common/src/main/java/com/wynntils/services/mapdata/MapAttributesResolver.java

```diff
@@ -66,33 +69,44 @@ private <T> T getAttribute(Function<MapAttributes, Optional<T>> attributeGetter)
         return attributeGetter.apply(DefaultMapAttributes.INSTANCE).get();
     }
 
+    private ResolvedMarkerOptions getResolvedMarkerOptions(
+            Function<MapAttributes, Optional<MarkerOptions>> attributeGetter) {
+        return new ResolvedMarkerOptions(
+                getInheritedValue(MarkerOptions::getInnerRadius, attributeGetter),
+                getInheritedValue(MarkerOptions::getOuterRadius, attributeGetter),
+                getInheritedValue(MarkerOptions::getFade, attributeGetter),
+                getInheritedValue(MarkerOptions::getBeaconColor, attributeGetter),
+                getInheritedValue(MarkerOptions::labelVisible, attributeGetter),
+                getInheritedValue(MarkerOptions::distanceVisible, attributeGetter),
```

Comment:
This still sounds like it's saying at what distance it becomes visible. Aligning with `hasIcon` and `hasLabel`, maybe `hasDistanceLabel`, or `hasDistanceDisplay`?